{"uuid": "772a5cfb-e2fd-4f14-aea9-1916155d92c5", "children": ["e33e3167-287d-4e1f-ba39-2dc924cbc521"], "befores": [{"name": "db_connection", "status": "skipped", "statusDetails": {"message": "Skipped: ❌ Database connection failed: connection to server at \"auroradb1-dev-staging-instance-1-eu-west-2b.callx7p4ov2n.eu-west-2.rds.amazonaws.com\" (10.4.21.227), port 3306 failed: received invalid response to SSL negotiation: J\n\n", "trace": "  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/tests/db/fixtures.py\", line 48, in db_connection\n    pytest.skip(f\"❌ Database connection failed: {e}\")\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/saravanakumar.veluchamy/Downloads/playwrightbddpython/venv/lib/python3.13/site-packages/_pytest/outcomes.py\", line 159, in skip\n    raise Skipped(msg=reason, allow_module_level=allow_module_level)\n"}, "start": 1760702072181, "stop": 1760702074372}], "start": 1760702072181, "stop": 1760702074381}